enterprise/server/backends/distributed/distributed.go

```diff
@@ -48,13 +52,19 @@ var (
 	newConsistentHashFunction    = flag.String("cache.distributed_cache.new_consistent_hash_function", "CRC32", "A consistent hash function to use when hashing data. CRC32 or SHA256")
 	newConsistentHashVNodes      = flag.Int("cache.distributed_cache.new_consistent_hash_vnodes", 100, "The number of copies of each peer on the new consistent hash ring")
 	newNodesReadOnly             = flag.Bool("cache.distributed_cache.new_nodes_read_only", false, "If true, only attempt to read from the newNodes set; do not write to them yet")
+
+	lookasideCacheSizeBytes = flag.Int64("cache.distributed_cache.lookaside_cache_size_bytes", 0, "If > 0 ; lookaside cache will be enabled")
+	lookasideCacheTTL       = flag.Duration("cache.distributed_cache.lookaside_cache_ttl", 15*time.Minute, "How long to hold stuff in the lookaside cache. Should be << atime_update_threshold")
 )
 
 const (
 	// Each hinted handoff is a digest (~64 bytes), prefix, and peer
 	// (40 bytes). So keeping around 100000 of these means an extra 10MB
 	// per peer.
 	maxHintedHandoffsPerPeer = 100000
+
+	// Max size of an entry that can be cached in the lookaside cache.
+	maxLookasideEntrySize = 100_000
```

Comment:
might be worth adding a flag for this for if we're flushing it way too fast

Reply:
Done.